packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -548,14 +545,13 @@ export class Interpreter {
 					tealVersion: this.tealVersion,
 				});
 			}
+			const costFromExecute = instruction.execute(this.stack);
+			const cost =
+				costFromExecute === undefined ? this.lineToCost[instruction.line] : costFromExecute;
```

Comment:
```suggestion
			const costFromExecute = instruction.execute(this.stack);
			const cost = costFromExecute ?? this.lineToCost[instruction.line];
```

Reply:
Should we return cost with all `instruction.execute()` ? Maybe we should do it in another PR.

Reply:
I will do it in next PR

Reply:
I added the cost for the rest of the opcodes